nano/lib/tomlconfig.cpp

```diff
@@ -186,6 +186,41 @@ void nano::tomlconfig::erase_default_values (tomlconfig & defaults_a)
 	erase_defaults (defaults_l.get_tree (), self.get_tree (), get_tree ());
 }
 
+// Merges two TOML configurations and commenting values that are identical
+std::string nano::tomlconfig::merge_defaults (nano::tomlconfig & current_config, nano::tomlconfig & default_config)
+{
+	auto default_tree = default_config.get_tree ();
```

Comment:
Why are default_tree and current_tree needed?

Reply:
You are right. It's not needed anymore. I'll change the assert to test the toml configs instead, and delete these lines